src/views/views.py

```diff
@@ -71,14 +72,19 @@ def home(request):
         order_by = filters.get('order_by') == 'asc' if '' else '-'
         job_offers = job_offers.order_by(f'{order_by}due_date')
 
+    if filters and filters.get('category'):
+        job_category_id = filters.get('category')
+        job_offers = Job.get_by_category(job_category_id)
```

Comment:
It's forbidden to access the Data layer from the UI layer, use controller

Reply:
wdym? I followed what is for the company and you don't complain about that